wordpressorg.dev/provision/wp-config.php

```diff
@@ -10,6 +10,8 @@
 define( 'API_WPORGPATH',          __DIR__ . '/../../api.wordpress.org/public_html/includes/' );
 define( 'GLOTPRESS_LOCALES_PATH', __DIR__ . '/../../translate.wordpressorg.dev/public_html/glotpress/locales/locales.php' );
 define( 'WPORG_SANDBOXED',        true );
+define( 'PLUGINS_TABLE_PREFIX',   $table_prefix );
+define( 'WP_CORE_STABLE_BRANCH',  '4.5' );
```

Comment:
Yeah, to the top of `sandbox-functionality.php`.

One of the goals of the Environment is to be as close to production as possible. `WP_CORE_STABLE_BRANCH` isn't defined in `wp-config` on production, and the file that it is defined in isn't open-sourced. The `sandbox-functionality` files are meant to be a place where we can do things that aren't done on production, or that are done on production but aren't open-sourced yet. That way there's a clean line between what's canonical code and what's just done to make things work in the Environment.